redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -135,6 +135,92 @@ PXE server with the network boot parameters specified in the request for the
 servers defined in the request. It then restarts these servers into one-time
 network boot, initiating the provisioning.
 
+==== Configuring PXE
+
+In {product-title}, a PXE server represents a collection of services and files
+that, all together, enable computers to boot into Operating Systems from the
+network:
+
+* The {product-title} needs to be able to connect to the PXE server for both
+  reading and writing using either NFS or SAMBA protocol. This is also called
+  the depot type.
+* The physical servers' NICs that are connected to the network boot network
+  require a TFTP access for reading the initial executable file containing the
+  PXE itself, and the HTTP access for downloading the Operating Systems' files.
+
+The process of configuring the PXE Servers is similar to that explained in the
+*PXE Provisioning* section of the *Provisioning Virtual Machines and Instances*
+chapter.
+
+We visit *Compute -> Infrastructure -> PXE*. Then, we click *Configuration ->
+Add a New PXE server*. We fill in the fields to describe the server.
+
+image::docs_redfish_using22_new_pxe_server_dialog.png[alt="New PXE Server dialog",512,align="center"]

```

Comment:
![image](https://user-images.githubusercontent.com/8102426/62940531-171cf180-bdd4-11e9-91fc-7037afdcc642.png)

I'm not sure PXE dir and Customization dir are supposed to be the same because in theory you could accidently name your customization template as `default` which would actually override your PXE menu definition.

Reply:
They are the same on our top-of-the-rack appliance setup. And the danger of overwriting an existing file is almost non-existent since customization templates that are put on the PXE server have names like `12:34:56:78:90:ab.cfg.ks`.

But I agree that maybe we should show give those two folders different names here since content of the first directory is expected to accessible over NFS/SAMBA and the customization directory over access url.